components/Ink/src/private/MDCInkLayer.m

```diff
@@ -73,28 +73,34 @@ static inline CGFloat MDCInkLayerRectHypotenuse(CGRect rect) {
 typedef NS_ENUM(NSInteger, MDCInkRippleState) {
   kInkRippleNone,
   kInkRippleSpreading,
-  kInkRippleComplete
+  kInkRippleComplete,
+  kInkRippleCancelled,
 };
 
 @protocol MDCInkLayerRippleDelegate <NSObject>
 
 @optional
 
-- (void)animationDidStop:(CAAnimation *)anim finished:(BOOL)finished;
+- (void)animationDidStop:(CAAnimation *)anim
+              shapeLayer:(CAShapeLayer *)shapeLayer
+                finished:(BOOL)finished;
 
 @end
 
 @interface MDCInkLayerRipple : CAShapeLayer
 
+@property (nonatomic, assign) BOOL animationsCleared;
```

Comment:
nit: Extra space before (

In order to be properly support KVC, BOOL properties should be declared:
@property(nonatomic, assign, getter=isAnimationCleared) BOOL animationCleared;

s/animationsCleared/animationCleared/ so the grammar doesn't get weird.
areAnimationsCleared wouldn't be KVC friendly.